app/src/main/kotlin/com/github/oryanmat/trellowidget/widget/TrelloWidgetProvider.kt

```diff
@@ -105,9 +107,10 @@ class TrelloWidgetProvider : AppWidgetProvider() {
         return PendingIntent.getActivity(context, appWidgetId, reconfigIntent, FLAG_IMMUTABLE)
     }
 
+    @SuppressLint("MutableImplicitPendingIntent")
```

Comment:
What's this for? I don't get a warning without it.

Reply:
Here is the warning I get:

> Mutable implicit PendingIntent will throw an exception once this app starts targeting Android 14 or above, follow either of these recommendations: for an existing PendingIntent use FLAG_NO_CREATE and for a new PendingIntent either make it immutable or make the Intent within explicit
Inspection info: Apps targeting Android 14 and above are not allowed to create PendingIntents with FLAG_MUTABLE and an implicit intent within for security reasons.  To retrieve an existing PendingIntent, use FLAG_NO_CREATE. To create a new PendingIntent, either make the intent explicit, or make it immutable with FLAG_IMMUTABLE.
Issue id: MutableImplicitPendingIntent

More in [the reference](https://developer.android.com/reference/android/app/PendingIntent).

I guess this PR is not the definitive solution, but a workaround to make things work as before. for new updates to SDK version we may need to make some changes to the app design.

Reply:
I guess that's fine for now then.

Reply:
It works on Android 15 / Pixel Pro 8.
So definitely a good solution for now.